www-cronrobot/Gemfile

```diff
@@ -64,7 +64,7 @@ group :development do
 end
 
 group :production do
-  gem 'pg'
+ gem 'pg'
```

Comment:
add an INFO box here to description quickly what resources are (background clue, with Info logo maybe).
I will probably only show it few times

![image](https://user-images.githubusercontent.com/8007722/132593545-8949331e-ea99-47c5-b603-d86aae8b1c0b.png)